src/Onboarding/FormRepository.php

```diff
@@ -66,17 +72,30 @@ protected function isFormAvailable($formID)
     }
 
     /**
+     * @unreleased Create the default v3 form.
      * @since 2.8.0
      * @return int Form ID
      *
      */
     protected function makeAndPersist()
     {
-        $formID = $this->defaultFormFactory->make();
+        $form = new DonationForm([
+            'title'    => __('GiveWP Donation Form', 'give'),
+            'status'   => DonationFormStatus::PUBLISHED(),
+            'settings' => FormSettings::fromArray([
+                'designId' => 'multi-step',
+                'designSettingsImageUrl' => GIVE_PLUGIN_URL . '/assets/dist/images/admin/onboarding/header-image.png',
```

Comment:
Remember to update it here too.

Reply:
Good catch, thanks! 2cf25936ee4ef5c9e6d3b6c41a4d009bce86de8c